include/circt/Dialect/HW/InnerSymbolTable.h

```diff
@@ -250,10 +250,10 @@ OS &operator<<(OS &os, const InnerSymTarget &target) {
     os << "field " << target.getField() << " of ";
 
   if (target.isPort())
```

Comment:
Please make this change separately.  Output change LGTM, especially for user-facing output/diagnostics!